src/tests/Directory.Build.props

```diff
@@ -8,6 +8,11 @@
   <Import Project="$(MSBuildThisFileDirectory)\Common\dir.sdkbuild.props" Condition="'$(UsingMicrosoftNETSdk)' == 'true'"  />
   <Import Project="$(MSBuildThisFileDirectory)\Common\dir.common.props" Condition="'$(UsingMicrosoftNETSdk)' != 'true'"  />
 
+  <PropertyGroup>
+    <!-- Override the compiler version with a private build that supports runtime-async -->
+    <MicrosoftNetCompilersToolsetVersion>5.0.0-1.25259.6</MicrosoftNetCompilersToolsetVersion>
+  </PropertyGroup>
```

Comment:
Is there a reason to have this in src/tests/Directory.Build.props instead of src/tests/async/Directory.Build.props?

Reply:
I don't think this worked when I tried it earlier. Let me try again.

Reply:
It needs to be here because we don't restore individual test projects (there's too many). Instead we restore the test_dependencies projects and point the individual test projects at those restore results.

One day I'll get this cleaned up I hope.